packages/core/typescript/TypeScriptFileUpdate.ts

```diff
@@ -211,7 +217,7 @@ export class TypeScriptFileUpdate {
 						...newProvides
 					]);
 
-					return ts.factory.updateArrayLiteralExpression(array, elements);
+					return ts.factory.updateArrayLiteralExpression(array, elements.map(x => x as ts.Expression));
```

Comment:
Not C#, looping over a collection to change the items type is a redundant operation, so please don't :)
You _can_ simply type assert `elements as ts.Expression[]` OR even better - how about just above when this is created we specify the item type there:
```ts
ts.factory.createNodeArray<ts.Expression>(...
```